packages/xchain-cosmos/src/util.ts

```diff
@@ -99,24 +107,22 @@ export const getTxsFromHistory = (txs: TxResponse[], mainAsset: Asset): Tx[] =>
         let to_index = -1
 
         to.forEach((value, index) => {
-          if (value.to === msgSend.to_address.toBech32()) to_index = index
+          if (value.to === msgSend.to_address.toString()) to_index = index
         })
 
         if (to_index === -1) {
           to.push({
-            to: msgSend.to_address.toBech32(),
+            to: msgSend.to_address.toString(),
             amount,
           })
         } else {
           to[to_index].amount = baseAmount(to[to_index].amount.amount().plus(amount.amount()), 6)
         }
       } else if (isMsgMultiSend(msg)) {
-        const msgMultiSend = msg as MsgMultiSend
+        const msgMultiSend = msg as proto.cosmos.bank.v1beta1.MsgMultiSend
```

Comment:
There should not be a need to use `as` if type guard `isMsgMultiSend(msg)` has been used before.

Reply:
fixed